BE/GovernmentEntities/AsianJurisdiction/CentralAsiaGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -0,0 +1,180 @@
+		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-casj;JurisdictionOfTheRepublicOfTajikistan"/>
+		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;Tajikistan"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-casj;GovernmentOfTheRepublicOfUzbekistan">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of the Republic of Uzbekistan</rdfs:label>
+		<rdfs:label xml:lang="uz">Oʻzbekiston Respublikasi</rdfs:label>
+		<skos:definition>unitary presidential constitutional secular republic in Central Asia, bordered by Kazakhstan to the north, Kyrgyzstan to the northeast, Tajikistan to the southeast, Afghanistan to the south, Turkmenistan and the autonomous republic of Karakalpakstan to the south-west</skos:definition>
+		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-casj;JurisdictionOfTheRepublicOfUzbekistan"/>
+		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;Uzbekistan"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-casj;GovernmentOfTurkmenistan">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of Turkmenistan</rdfs:label>
+		<rdfs:label xml:lang="tk">Türkmenistan</rdfs:label>
+		<skos:definition>unitary dominant-party presidential republic in Central Asia, bordered by Kazakhstan to the northwest, Uzbekistan to the north, east and northeast, Afghanistan to the southeast, Iran to the south and southwest and the Caspian Sea to the west</skos:definition>
+		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-casj;JurisdictionOfTurkmenistan"/>
+		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;Turkmenistan"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-casj;JurisdictionOfTheKyrgyzRepublic">
+		<rdf:type rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
+		<rdfs:label>jurisdiction of the Kyrgyz Republic</rdfs:label>
+		<skos:definition>jurisdiction of the judiciary of Kyrgyzstan, which is the system of courts, including the Supreme Court, that interprets and applies the law in Kyrgyzstan</skos:definition>
+		<fibo-be-ge-ge:isJurisdictionOf rdf:resource="&fibo-be-ge-casj;GovernmentOfTheKyrgyzRepublic"/>
+		<fibo-fnd-law-jur:hasReach rdf:resource="&lcc-3166-1;Kyrgyzstan"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-casj;JurisdictionOfTheRepublicOfKazakhstan">
+		<rdf:type rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
+		<rdfs:label>jurisdiction of the Republic of Kazakhstan</rdfs:label>
+		<skos:definition>jurisdiction of the judiciary of Kazakhstan, which is the system of courts that interprets and applies the law in Kazakhstan</skos:definition>
+		<fibo-be-ge-ge:isJurisdictionOf rdf:resource="&fibo-be-ge-casj;GovernmentOfTheRepublicOfKazakhstan"/>
```

Comment:
It seems redundant to populate both directions of the pair of inverse properties; and bad practice in terms of maintenance (two triples would need to be updated). isJurisdictionOf seems the unnecessary one.